guacamole-common-js/src/main/webapp/modules/Client.js

```diff
@@ -1824,3 +1844,31 @@ Guacamole.Client.DefaultTransferFunction = {
     }
 
 };
+
+/**
+ * A list of possible messages that can be sent by the server and displayed to
+ * the client.
+ * 
+ * @type {!Object.<string, number>}
+ */
+Guacamole.Client.Message = {
+    
+    /**
+     * A client message that indicates that a user has joined an existing
+     * connection. This message xpects a single additional argument - the
```

Comment:
Typo: `xpects`

Reply:
That's the hip (read: lazy SMS) way to spell it.

Fixd.